contrib/build-push-ceph-container-imgs.sh

```diff
@@ -17,14 +17,34 @@ function login_docker_hub {
   docker login -u "$DOCKER_HUB_USERNAME" -p "$DOCKER_HUB_PASSWORD"
 }
 
+function create_head_or_point_release {
+  local latest_tag
+  local latest_commit_sha
+  local current_branch
+  # we test if we are running on a tagged commit
+  # if so, we build images with this particular tag
+  # otherwise we just build using the branch name (make's default)
+  latest_tag=$(git describe --exact-match HEAD --tags --long 2>/dev/null)
+  if [ "$?" -eq 0 ]; then
+    echo "Building a release Ceph container image based on tag $latest_tag"
+    RELEASE=( "RELEASE=$latest_tag" )
+  else
+    echo "Building a devel Ceph container image based on branch $current_branch and commit $latest_commit_sha"
+    latest_commit_sha=$(git rev-parse --short HEAD)
+    current_branch=$(git rev-parse --abbrev-ref HEAD)
+    RELEASE=( "RELEASE=$current_branch-$latest_commit_sha" )
```

Comment:
Why such complexity ?
Why not having RELEASE="something"  and later on doing a  "make "$RELEASE" build.parallel "

I have the feeling I missed something here but I cannot get what.

Reply:
In our makefile, RELEASE is already et to the branch name if no RELEASE is given.
Why not keeping that default behavior ?